composer/packages/api-designer/src/api-designer/components/header/header.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from "react";
```

Comment:
Shall we add license headers for all the newly added files?

Reply:
Fixed with commit 4d0cc36